diffgraph/cli.py

```
@@ -91,39 +53,35 @@ def get_changed_files() -> List[Dict[str, str]]:
+                changed_files.append({
+                    'path': file_path,
+                    'status': 'untracked'
+                })
+        except subprocess.CalledProcessError as e:
+            click.echo(f"Error getting untracked files: {e}", err=True)
+            sys.exit(1)
 
     return changed_files
 
-def load_file_contents(changed_files: List[Dict[str, str]]) -> List[Dict[str, str]]:
+def load_file_contents(changed_files: List[Dict[str, str]], diff_args: List[str] = []) -> List[Dict[str, str]]:
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)